tests/mul.rs

```diff
@@ -0,0 +1,16 @@
+#[cfg(test)]
+mod tests {
+    use lrlex::lrlex_mod;
+    use lrpar::lrpar_mod;
+    use yaiwr::eval;
+    lrlex_mod!("calc.l");
+    lrpar_mod!("calc.y");
+    #[test]
```

Comment:
Maybe a newline before this line, but up to you.

Reply:
adedd